subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -140,6 +140,9 @@ static void prio_recv_thread(void *p1, void *p2, void *p3)
 
 			buf = process_prio_evt(node_rx);
 			if (buf) {
+				node_rx->hdr.next = NULL;
```

Comment:
#if defined(CONFIG_BT_LL_SW_LEGACY)
{
		extern u8_t radio_rx_fc_set(u16_t handle, u8_t fc);

		radio_rx_fc_set(node_rx->hdr.handle, 0);
	}
#endif /* CONFIG_BT_LL_SW_LEGACY */


Add this code before the release for legacy controller